lib/cosi/cosi.go

```diff
@@ -0,0 +1,236 @@
+package cosi
+
+import (
+	"errors"
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/config"
+	"time"
+)
+
+// Cosi is the struct that implements the "vanilla" cosi.
+type Cosi struct {
+	// Suite used
+	suite abstract.Suite
```

Comment:
Yes, but as you say, *for the moment*, so it won't have to be changed later on. I take ```node.Suite() ```, which most protocols will do anyway because they don't care (and should not care) about the network.